src/net/sourceforge/zmanim/hebrewcalendar/JewishCalendar.java

```diff
@@ -436,6 +436,15 @@ public boolean isErevYomTovSheni() {
 				|| (getJewishMonth() == JewishCalendar.SIVAN && getJewishDayOfMonth() == 6 )));
 	}
 
+	/**
+	 * Returns true if the current day is <em>Aseret Yemei Teshuva</em>.
+	 * 
+	 * @return if the current day is <em>Aseret Yemei Teshuvah</em>
+	 */
+	public boolean isAseretYemeiTeshuva(){

```

Comment:
@plonibarploni since I use Ashkenazi transliteration throughout the code, can you change this to isAseresYemeiTeshuva()?
Thanks

Reply:
Fixed